doc/source/dev/contributor/contributor_toc.rst

```diff
@@ -0,0 +1,94 @@
+- :ref:`docs` - how to write reStructuredText documentation that complies with docstring standards, build documentation locally with Sphinx, and view documentation built during continuous integration checks
+- :ref:`toc-benchmarking` - how to benchmark code with AirSpeed Velocity
+- :ref:`toc-cython` - how to add fast, compiled code to SciPy
+
+.. _dev-env:
+
+Development Environment
+-----------------------
+- :ref:`quickstart-mac` presents a step-by-step process for setting up a convenient SciPy development environment in macOS.
+- :ref:`building` - If you don't have macOS, try these instructions to help you build SciPy on your operating system.
+- :ref:`recommended-development-setup` includes additional notes about the development setup. All of this information is contained elsewhere, but it is retained as a legacy document.
+
+.. _editing-scipy:
+
+Editing SciPy
+-------------
+- :ref:`development-workflow` lays out what to do after your development environment is set up.
+- `SciPy Development Workflow`_ is a five-minute video example of fixing a bug and submitting a pull request.
+- :ref:`pep8-scipy` gives some tips for ensuring that your code is PEP8 compliant.
+- :ref:`git-development` is a guide to using ``git``, the distributed version-control system used to manage the changes made to SciPy code from around the world.
+- :ref:`scipy-api` contains some important notes about how SciPy code is organized and documents the structure of the SciPy API. If you are going to import other SciPy code, read this first.
+- `NumPy Distutils - Users Guide`_ - Check this out before adding any new files to SciPy.
+- :ref:`core-dev-guide` has background information including how decisions are made and how a release is prepared. It's geared toward :ref:`Core Developers<governance>`, but contains useful information for all contributors.
+
+.. _unit-tests:
+
+Unit Tests
+----------
+- `NumPy/SciPy Testing Guidelines`_ is the definitive guide to writing unit tests of SciPy code.
+- :ref:`runtests` documents ``runtests.py``, a convenient script for building SciPy and running tests locally.
+
+.. _docs:
+
+Documentation
+-------------
+- `numpydoc docstring guide`_ contains everything you need to know about writing docstrings, which are rendered to produce HTML documentation using `Sphinx`_.
+- :ref:`rendering-documentation` - It's important to check how changes to the documentation render before merging a PR; this document explains how you can do that.
+
+.. _toc-benchmarking:
+
+Benchmarks
+----------
+- :ref:`benchmarking-with-asv` explains how to add benchmarks to SciPy using `Airspeed Velocity`_.
+
+
+.. _toc-cython:
+
+.. _compiled-code:
+
+Compiled Code
+-------------
+- :ref:`adding-cython` - Extending and compiling Python code with `Cython`_ can significantly improve its performance. This document helps you get started.
+- :ref:`other-languages` discusses the use of C, C++, and Fortran code in SciPy.
+
+Here is a toctree, purely to appease Sphinx.
```

Comment:
You can use `:orphan:` at the top of the files that do not belong to a toctree / that you link to manually.

Let me know when you push and CircleCI is happy and I'll look

Reply:
Done